components/Header/Header.tsx

```diff
@@ -25,29 +37,26 @@ const Header: React.FC = () => {
           <li className="flex items-center gap-2 cursor-pointer body-roman">
             <NavDropdown />
           </li>
-          <li className="cursor-pointer body-roman">
-            <Link href={LINKS.PROJECTS} className='transition duration-300 group'>הפרויקטים
-            <span className="block max-w-0 group-hover:max-w-full transition-all duration-500 h-0.5 bg-sky-600"/>
+          <li className={`cursor-pointer body-roman ${currentPage === 'projects' ? 'font-bold' : ''}`}>
```

Comment:
Maybe using an array for future use if there will be changes in the nav. All the styles here is duplicated

Reply:
> Maybe using an array for future use if there will be changes in the nav. All the styles here is duplicated

makes sense....want me to assign you this?

Reply:
@Tamir198  Is already on it no?

Reply:
He did start but was not assigned its an oversight on my end.
he is now :D
thanks for looking into this!

Reply:
Do you mean an array that will contain the strings? something else? 

Generally, we have a lot of styles duplicated all over the project.
In my opinion, It will be better to use [layer](https://tailwindcss.com/docs/functions-and-directives) to create custom classes for generic purposes (for example, card, navItem etc...) together with generic components.

If the same style repeats itself it can be a good indication that this part can be extracted into a custom component. 

I think that think is a project level thing, related to the project architecture.

Reply:
I was thinking about array of the items in the header and then iterate over it inside the header with the same style